File: MarcosSenn/Python/Clase5/ejercicio1.py
#Ejercicio 1: Sumar numeros pares dentro de un rango

num1 = int(input('Ingrese un numero'))
num2 = int(input('Ingrese otro numero'))
print('Los numeros pares dentro de ese rango son:')

for i in range(num1,num2):
    if(i%2==0):
        print(i, end=' ')

File: MarcosSenn/Java/Clases/src/Clase11/Ejercicio1.java
package ejercicios;

import java.util.Scanner;

public class Ejercicio1 {
    public static void main(String[] args) {
        int numeros[] = new int[5];
        Scanner sc = new Scanner(System.in);

        System.out.println("Ingrese 5 numeros: ");
        for(int i = 0; i < numeros.length; i++){
            System.out.println("Numero " + (i+1) + ": ");
            numeros[i] = sc.nextInt();
        }

        System.out.println("Los numeros cargados son: ");
        for(int i = 0; i < numeros.length; i++){
            System.out.println("Numero " + (i+1) + ": " + numeros[i]);
        }
    }
}






